src/module/combat.js

```diff
@@ -344,5 +353,7 @@ export class OseCombat {
     } else if (init === "individual") {
       OseCombat.individualInitiative(combat, data, diff, id);
     }
-  }
-}
+  },
+};
+
+export default OseCombat;
```

Comment:
That's the weird thing here, it's converting the class into an object (which, I mean, all JS classes *are* objects). I think it's because everything in the class was static.